content/en/security_platform/application_security/setup_and_configure.md

```diff
@@ -292,11 +292,51 @@ if span, ok := tracer.SpanFromContext(request.Context()); ok {
 
 {{< programming-lang lang="ruby" >}}
 
+Use either API to add user information to a trace so that you can monitor authenticated requests in the application.
+
+{{< tabs >}}
+
+{{% tab "Using `Datadog::Kit::Identity.set_user` %}}
+
+Starting with `ddtrace` 1.1.0, a convenience `set_user` method is available:
+
+```ruby
+# Get the active trace
+trace = Datadog::Tracing.active_trace
+
+# Set mandatory user id tag
+Datadog::Kit::Identity.set_user(trace, id: 'd131dd02c56eeec4')
+
+# Or set any of these optional user monitoring tags
+Datadog::Kit::Identity.set_user(
+  trace,
+
+  # mandatory id
+  id: 'd131dd02c56eeec4',
+
+  # optional tags with known semantics
+  name: 'Jean Example',
+  email:, 'jean.example@example.com',
+  session_id:, '987654321',
+  role: 'admin',
+  scope: 'read:message, write:files',
+
+  # optional free-form tags
+  another_tag: 'another_value',
+)
+```
+
+{{% /tab %}}
+
+{{% tab "Using trace `set_tag`" %}}
```

Comment:
Less of a correction issue, but more of a documentation clarity issue: we seem to be giving users a lot of alternatives to do things: two ways of onboarding for each variant, two ways of setting tags, etc etc.

I wonder if we should be a bit more opinionated and just state one option? We could always serve advanced users using the API docs or some other advanced doc. See for instance <https://documentation.divio.com/introduction/#the-secret> for a discussion of this.